server/models/database.js

```diff
@@ -1,21 +1,28 @@
 const DB_USER = process.env.DB_USER;
 const DB_USER_PASSWORD = process.env.DB_USER_PASSWORD;
 
-const DB_NAME =
-  process.env.NODE_ENV === "test"
-    ? process.env.DATABASE_TEST
-    : process.env.DATABASE;
-
-let connectionURI;
-if (DB_USER && DB_USER_PASSWORD && process.env.NODE_ENV !== "test")
-  connectionURI = `mongodb+srv://${DB_USER}:${DB_USER_PASSWORD}@cluster0.kbmrd.mongodb.net/grassHeap?retryWrites=true&w=majority`;
-else connectionURI = `mongodb://localhost:27017/${DB_NAME}`;
-
-const mongoose = require("mongoose");
-mongoose.connect(connectionURI, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
+function makeConnectionURI() {
+  const IS_TEST = process.env.NODE_ENV === 'test';
+  const DB_NAME = IS_TEST ? process.env.DATABASE_TEST : process.env.DATABASE;
+
+  if (!DB_USER || !DB_USER_PASSWORD || IS_TEST) {
+    return `mongodb://localhost:27017/${DB_NAME}`;
+  }
+
+  return `mongodb+srv://${DB_USER}:${DB_USER_PASSWORD}@cluster0.9g4e8.mongodb.net/${DB_NAME}?retryWrites=true&w=majority`;
```

Comment:
will using a different URI (uses '9g4e8' vs 'kbmrd') not prevent connection the currently used database which holds all the tasks? Maybe this part of the URI should also be dynamic?

Reply:
You are absolutely right, I forgot and commited that part as well.

It should come from the .env as well, maybe as `DB_CLUSTER` or something

Reply:
I've added `DB_HOST` to the `.env` file.

Remember to update environment settings on **Heroku** when it's deployed.

Also: moved **Prettier** config to the root folder so it's shared between client and server 😄